### PR TITLE
add BlogApp controller rspec

### PR DESCRIPTION
**Pull request by Ellyboi**
**Hello @Code Reviewer 👋  Kindly check my project for both positive and negative criticism, i believe it will be positive though 😆** 

**Project requirements**
- Add RSpec as a gem to your project.
- Create a Request spec file for all controllers.
- For each action I check:
        - response status was correct.
        - correct template was rendered.
        - the response body includes correct placeholder text.

speccontroller

### CODE REVIEW

File: spec/posts_controller_spec.rb [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+  describe 'GET #show' do
+    it 'returns http success' do
+      get user_post_path(user_id: 1, id: 1)
+
+      expect(response).to have_http_status(:success)
+      expect(response).to render_template(:show)
+    end
+  end
+end
```
Comment (mckent05) [inline spec/posts_controller_spec.rb]: - Good job done so far, but according to the project requirements, you are required to check if the response body includes the correct placeholder text. Please fix this 😄. For example to check if the `post#show` response body includes the correct placeholder test:
```ruby
expect(response.body).to include("Post Details")
```

File: spec/users_controller_spec.rb [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+  describe 'GET #show' do
+    it 'returns http success' do
+      get user_path(1)
+
+      expect(response).to have_http_status(:success)
+      expect(response).to render_template(:show)
+    end
+  end
+end
```
Comment (mckent05) [inline spec/users_controller_spec.rb]: - Good job done so far, but according to the project requirements, you are required to check if the response body includes the correct placeholder text. Please fix this for both actions 😄. For example to check if the `user#show` response body includes the correct placeholder test:
```ruby
expect(response.body).to include("User Details")
```

**Review by mckent05 — CHANGES_REQUESTED** [review]
Hi @Ellyboi, 

# STATUS: CHANGES REQUIRED ♻️ ♻️ 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
almost there

## Highlights
- All linter checks are passing ✔️ ✔️ 
- Your PR is professional 👍 👍 
- Good job testing your controllers 👍 👍 


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by ridwanediallo — APPROVED** [review]
# STATUS: APPROVED 🟢
## Hi @Ellyboi , 
Congratulations!GIF

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## (Highlights) Good Points: :+1:
- You documented your work professionally. ✔
- Well done implementing the the changes requested by the previous reviewer. ✔
- You successfully completed all the requirements.  🎉 🎉 🎉 ✔

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**